components/thumbnail/metadata/thumbnail.yml

```diff
@@ -3,20 +3,8 @@ status: Contribution
 sections:
   - name: Migration Guide
     description: |
-      ### T-shirt sizing
-      Thumbnail now supports t-shirt sizing with the standard naming convention and requires that you specify the size by adding a `.spectrum-Thumbnail--size*` class.
-
-      Additionally, the available sizes have changed: `XXL` and `XL` have been completely removed, and `XS` and `XXS` have been added.
-
-      | Deprecated Classname | New Classname           |
-      | -------------------- | ----------------------- |
-      | `~.spectrum-Thumbnail--XXL~` | REMOVED |
-      | `~.spectrum-Thumbnail--XL~` | REMOVED |
-      | `~.spectrum-Thumbnail--L~` | `.spectrum-Thumbnail--sizeL` |
-      | `~.spectrum-Thumbnail--M~` | `.spectrum-Thumbnail--sizeM` |
-      | `~.spectrum-Thumbnail--S~` | `.spectrum-Thumbnail--sizeS` |
-      | | `.spectrum-Thumbnail--sizeXS` |
-      | | `.spectrum-Thumbnail--sizeXXS` |
+      ### Thumbnail sizing
+      Thumbnail sizes are specified by appending a number to the size class: `.spectrum-Thumbnail--size*`.
```

Comment:
Should we specify what those number options are? That might be overkill but I don't know how knowledgable the developers using this are of their options

Reply:
thats a good call, added language from design documentation.